security.md

```diff
@@ -80,6 +81,15 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2020-14334"></a>CVE-2020-14334: World readable cache could expose sensitive settigs
+Even encrypted settings have their raw values cached. Too permissive mode on cache dir caused, that anyone with access to the hosting system could read this encrypted settings.
+
+*Mitigation:* change cache directory mode `o-r` on `/run/foreman`.
```

Comment:
This is not a correct mitigation because it's lost at reboot. You must manually change the tmpfiles as well.

Reply:
Change that to advise the temfile rewrite from `/etc/tmpfiles.d`